tests/lib/pkgdb.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+. $TESTSLIB/quiet.sh
+
+debian_name_package() {
+    case "$1" in
+        xdelta3)
+            ;&
```

Comment:
It marks a fall through. See https://stackoverflow.com/questions/12010686/case-statement-fallthrough